website/src/views/JoinClass/JoinClass.css

```diff
@@ -64,17 +72,22 @@
   padding: 20px 0;
 }
 
-.add-distance {
-  margin-top: 5px !important;
+.join-class-page .add-distance {
+  margin-top: 5px;
 }
 
-.join-class-label {
+.join-class-page .join-class-label {
   margin: 0;
-  margin-top: 15px !important;
-  margin-bottom: 5px !important;
-  font-size: 16px !important;
+  margin-top: 15px;
+  margin-bottom: 5px;
+  font-size: 16px;
 }
 
 .join-class-label span {
   font-weight: 600;
 }
+
+.join-class-page .ant-table-thead > tr > th,
+.join-class-page .ant-table-tbody > tr > td {
```

Comment:
what does this override?

Reply:
This overrides the styling in the join class table which caused words to appear jumble on smaller screens do to word clipping